src/main/scala/org/zalando/spearheads/innkeeper/utils/HttpClient.scala

```diff
@@ -45,12 +45,12 @@ class AkkaHttpClient @Inject() (implicit val actorSystem: ActorSystem,
 
     for {
       jsonString <- Try {
-        logger.debug(s"We call the service with the url: ${uri}")
+        logger.info(s"We call the service with the url: ${uri}")
```

Comment:
that should remain debug... that is not info-level information... that is purely debug-relevant